### DISCUSSION TITLE
[feature request] CVS modified lines

### DISCUSSION THREAD

**Discussion by arogozhnikov** [microsoft/monaco-editor#3634] [Q&A]
Can we have this beautiful VS code hinting of which lines were changed in monaco? (But not invoking diff editor)

![image from https://stackoverflow.com/questions/41219116/vs-code-highlight-modified-lines](https://i.stack.imgur.com/MBIqc.jpg)

**Comment by KaKi87** [comment]
Hello @arogozhnikov, did you found any other code editor which has this feature so I could have something for the time being ? Thanks

**Comment by arogozhnikov** [comment]
@KaKi87 nope, sorry. Wasn't looking too vigorously though :)

**Comment by KaKi87** [comment]
@hediet Any news on this ? By the way, it's a feature request, not a question. Thanks
